### DISCUSSION TITLE
Quark Engine is here to help!

### DISCUSSION THREAD

**Discussion by 18z** [Pithus/bazaar#3] [General]
Hi @Pithus 

This is KunYu, founder of quark-engine.
I just saw that bazaar is using quark as one of its analyzers.

We'd like to know and help if there's any problem while integrating quark?
Please let us know if you need supports! 

You are welcomed to join our public telegram channel to chat with us! 😃 
https://t.me/joinchat/GKm-gRGj80-Zz3jX

KunYu

**Comment by U039b** [comment]
Hi @18z 
Thank you for your message and for awesome engine. I faced no issue while integrating quark :heart: 

In a near future I plan to:
- improve the UI of the quark report (show call graphs...)
- add metadata to the rules in order to compute a matrix of data sources *vs.* data sinks (SMS are collected and sent to a remote actor over UDP...)
- make rules searchable (get all samples collecting SMS and sending them over UDP)
